data/publiccloud/terraform/azure_nfstest.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     azurerm = {
-      version = ">= 3.2.0"
+      version = "= 3.2.0"
```

Comment:
I am not sure what was your intention . this change basically downgrade azurerm to version 3.2.0 . I don't think that it is good idea . Currently failed run was using 3.72 latest passed was on 3.70 so as hotfix we may rollback to THAT version

Reply:
Is there "new format"? And is there even any significant change in the provider affecting us?
Or can we just upgrade, test it, maybe do major tweak of our existing files and be done with it?

Reply:
I can not answer this questions without running test run . this is why we have version of provider locked here because from time terraform providers changing their format so our old profiles will start to fail . We have two options here : 
1. remove this artificial limitation and learn about this changes from mass failures of all Azure related tests . So we will need to firefight the issue in urgent mode .
2. Keep version locked and do from time to time "upgrade cycles" which includes test runs on the latest provider version 

I am prefer second option

Reply:
Yes, me too. We chose the second option.
I just wanted to say that we should not explicitly see upgrading as problem.
We should upgrade and do some verification runs before merging so we're sure.